### PR TITLE
Added CI

### PR DESCRIPTION
**Pull request by sabre1041**
Added CI

### CODE REVIEW

File: roles/common_criteria_tomcat/tasks/valide_context_xml.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
-- include_task: tasks/checks_conf_files.yml
-  with_items:
-    - { path: "{{ item }}", owner: "{{ tomcat_owner }}", group: "{{ tomcat_group }}", mode:
+# - include_task: tasks/checks_conf_files.yml
+#   with_items:
+#     - { path: "{{ item }}", owner: "{{ tomcat_owner }}", group: "{{ tomcat_group }}", mode:
```
Comment (rpelisse) [inline roles/common_criteria_tomcat/tasks/valide_context_xml.yml:4]: @sabre1041 Is still supposed to be commented? If so, I would rather add a "when:" statement so we can execute or not if we want.

File: roles/tomcat/tasks/main.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   assert:
     that:
       - tomcat.home is defined
-      - tomcat.home != ""
+      - tomcat.home | length > 0
```
Comment (rpelisse) [inline roles/tomcat/tasks/main.yml:6]: @sabre1041  :+1:

File: roles/common_criteria_tomcat/tasks/valide_context_xml.yml (hunk 1 above)
Comment (sabre1041) [reply roles/common_criteria_tomcat/tasks/valide_context_xml.yml:4]: @rpelisse is this actually being called?

Comment (rpelisse) [reply roles/common_criteria_tomcat/tasks/valide_context_xml.yml:4]: Then it's an unfinished file that needs to be remove. I'll take care of it after merging your changes (when ready)

**Comment by sabre1041** [comment]
@rpelisse should be good to go!

**Pull request merged**